pootle/apps/pootle_profile/utils.py

```diff
@@ -18,6 +23,18 @@ def __init__(self, user):
         self.user = user
 
     @cached_property
+    def avatar(self):
+        return render_as_template(
+            "{% load common_tags %}{% avatar username email_hash 20 %}",
```

Comment:
i dont really see a problem with it - altho it does feel something of an antipattern - its more efficient than using another inclusion tag i think